crates/biome_js_analyze/src/lint/nursery/no_secrets.rs

```diff
@@ -0,0 +1,287 @@
+        min_len: 32,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&SLACK_WEBHOOK_REGEX),
+        comment: "Slack Webhook",
+        min_len: 24,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&GITHUB_TOKEN_REGEX),
+        comment: "GitHub",
+        min_len: 35,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&TWITTER_OAUTH_REGEX),
+        comment: "Twitter OAuth",
+        min_len: 35,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&FACEBOOK_OAUTH_REGEX),
+        comment: "Facebook OAuth",
+        min_len: 32,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&GOOGLE_OAUTH_REGEX),
+        comment: "Google OAuth",
+        min_len: 24,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&AWS_API_KEY_REGEX),
+        comment: "AWS API Key",
+        min_len: 16,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&HEROKU_API_KEY_REGEX),
+        comment: "Heroku API Key",
+        min_len: 12,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&PASSWORD_IN_URL_REGEX),
+        comment: "Password in URL",
+        min_len: 14,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&GOOGLE_SERVICE_ACCOUNT_REGEX),
+        comment: "Google (GCP) Service-account",
+        min_len: 14,
+    },
+    SensitivePattern {
+        pattern: Pattern::Regex(&TWILIO_API_KEY_REGEX),
+        comment: "Twilio API Key",
+        min_len: 32,
+    },
+    SensitivePattern {
+        pattern: Pattern::Contains("-----BEGIN RSA PRIVATE KEY-----"),
+        comment: "RSA Private Key",
+        min_len: 64,
+    },
+    SensitivePattern {
+        pattern: Pattern::Contains("-----BEGIN OPENSSH PRIVATE KEY-----"),
+        comment: "SSH (OPENSSH) Private Key",
+        min_len: 64,
+    },
+    SensitivePattern {
+        pattern: Pattern::Contains("-----BEGIN DSA PRIVATE KEY-----"),
+        comment: "SSH (DSA) Private Key",
+        min_len: 64,
+    },
+    SensitivePattern {
+        pattern: Pattern::Contains("-----BEGIN EC PRIVATE KEY-----"),
+        comment: "SSH (EC) Private Key",
+        min_len: 64,
+    },
+    SensitivePattern {
+        pattern: Pattern::Contains("-----BEGIN PGP PRIVATE KEY BLOCK-----"),
+        comment: "PGP Private Key Block",
+        min_len: 64,
+    },
+];
+
+static mut MIN_PATTERN_LEN: Option<usize> = None;
+static INIT: Once = Once::new();
+
+fn get_min_pattern_len() -> usize {
+    unsafe {
+        INIT.call_once(|| {
+            MIN_PATTERN_LEN = Some(
+                SENSITIVE_PATTERNS
+                    .iter()
+                    .map(|pattern| pattern.min_len)
+                    .min()
+                    .unwrap_or(0),
+            );
+        });
+        MIN_PATTERN_LEN.unwrap_or(0)
+    }
+}
+
+fn is_high_entropy(text: &str) -> bool {
+    let entropy = calculate_shannon_entropy(text);
+    entropy > 4.5 // TODO: Make this optional, or controllable
```

Comment:
Could you use a constant instead of a inline value?

Something like:

```suggestion
    entropy > HIGH_ENTROPY_THRESHOLD // TODO: Make this optional, or controllable
```

Reply:
Done!